transform-jsonata/ssl/server.crt

```diff
@@ -0,0 +1,21 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
wont these be injected

Reply:
Yes, this is just for testing / local dev